vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/DragAndDropSteps.java

```diff
@@ -59,9 +66,47 @@ public DragAndDropSteps(IWebDriverProvider webDriverProvider, WebJavascriptActio
      * @param location location relatively to the <b>target</b> element (<b>TOP</b>,<b>BOTTOM</b>,<b>LEFT</b>,
      * <b>RIGHT</b>,<b>CENTER</b>,<b>LEFT_TOP</b>,<b>RIGHT_TOP</b>,<b>LEFT_BOTTOM</b>,<b>RIGHT_BOTTOM</b>)
      * @param target target element
+     * @deprecated Use step:
+     * "When I drag element located by `$draggable` and drop it at $location of element located by `$target`" instead
      */
+    @Deprecated(since = "0.6.4", forRemoval = true)
```

Comment:
```suggestion
    @Deprecated(since = "0.6.5", forRemoval = true)
```
`0.6.4` is released, it's needed to update `@since` version to `0.6.5` everywhere

Reply:
btw it's a good example of one of the reasons why it's not recommended to delay with PR fixes in general